galaxy/api/views/views.py

```diff
@@ -109,6 +117,71 @@ def filter_rating_queryset(qs):
 # -----------------------------------------------------------------------------
 
 
+class MonitorRootView(base_views.APIView):
+    """ Monitor resources """
+    permission_classes = (AllowAny,)
+
+    monitor_response = OrderedDict()
+
+    def get(self, request, format=None):
+        if not settings.GALAXY_METRICS_ENABLED:
+            self.monitor_response['influx'] = 'n/a'
+        else:
+            influxdb_client = influxdb.InfluxDBClient(
+                host=settings.INFLUX_DB_HOST,
+                port=settings.INFLUX_DB_PORT,
+                username=settings.INFLUX_DB_USERNAME,
+                password=settings.INFLUX_DB_PASSWORD
+            )
+
+            try:
+                influxdb_client.ping()
+                self.monitor_response['influx'] = 'ok'
+            except influxdb.client.InfluxDBClientError:
+                self.monitor_response['influx'] = 'error'
+
+            influxdb_client.close()
+
+        try:
+            connection = connections[DEFAULT_DB_ALIAS]
+            connection.prepare_database()
+            executor = MigrationExecutor(connection)
+            targets = executor.loader.graph.leaf_nodes()
+
+            if not executor.migration_plan(targets):
+                self.monitor_response['migrations'] = 'ok'
+            else:
+                self.monitor_response['migrations'] = 'needed'
+
+            self.monitor_response['postgresql'] = 'ok'
+        except DatabaseError:
+            self.monitor_response['postgresql'] = 'error'
+
+        redis_client = redis.Redis(
+            host=settings.REDIS_HOST,
+            port=settings.REDIS_PORT,
+            db=0
```

Comment:
Why `db=0` is hardcoded here? It's a default value for `redis.Redis` client.

Reply:
I'll remove this line.